docs/results.md

```diff
@@ -0,0 +1,376 @@
+
+## Host directories
+The results directory contains sub-directories for each PE infrastructure host.
+In the Standard reference architecture example above this would be the PE 'master'.
+A sub-directory for the 'metric' node (where the Gatling simulation is run) is also included.
+The name of each sub-directory corresponds to the hostname of the node.
+
+### Master
+#### atop measurements
+The 'master' node directory contains the atop measurements provided by [beaker-benchmark](https://github.com/puppetlabs/beaker-benchmark) and additional CSV and HTML files generated from these files.
+For example:
+```
+results/perf/PERF_1572446968/ip-10-227-0-214.amz-dev.puppet.net
+├── atop_log_applestoapples_json.csv
+├── atop_log_applestoapples_json.detail.csv
+├── atop_log_applestoapples_json.detail.csv.html
+├── atop_log_applestoapples_json.log
+├── atop_log_applestoapples_json.log.tar.gz
+├── atop_log_applestoapples_json.summary.csv
+├── atop_log_applestoapples_json.summary.csv.html
+├── atop_log_warmupjit_json.log
+└── atop_log_warmupjit_json.log.tar.gz
+```
+
+In the above example the `atop_log_warmupjit_json.log` and `atop_log_applestoapples_json.log` files are the full atop logs produced by beaker-benchmark.
+The 'warmupjit' log is created during the 'warm-up' portion of the test run; the 'applestoapples' log is created during the main portion of the test run.
+Due to potentially large log file sizes beaker-benchmark compresses these files before copying to the test runner; these are the corresponding `tar.gz` files.
+
+The `atop_log_applestoapples_json.csv` file is a CSV file containing data extracted from the corresponding log file.
+It contains a summary of the measurements along with per-process measurements.
+However, the formatting of this file is technically invalid due to a mismatched number of columns per row between these two sections.
+
+As a workaround for this formatting issue the file is split into two separate CVS files: `atop_log_applestoapples_json.summary.csv` and `atop_log_applestoapples_json.detail.csv`.
+This allows each section to be processed separately, including conversion to HTML documents via the [`csv2html`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/7278352a6626e1f22ae8d17a7d36aa36c94b5cf4/tests/helpers/perf_results_helper.rb#L236) utility.
+The HTML documents use a basic [Bootstrap](https://getbootstrap.com/) template which provides a clean layout for standalone viewing,
+and allows the tables to be extracted for inclusion in larger reports.
+
+#### var logs
+TBD: [SLV-691](https://tickets.puppetlabs.com/browse/SLV-691) has been created to update this document to include the logs once the directory has been archived before copying to the results.
+
+### Metric
+#### Gatling results
+The 'metric' node directory contains the Gatling results from the test run. 
+These results are contained within a sub-directory of the `root/gatling-puppet-load-test/simulation-runner/results` directory.
+For example:
```

Comment:
I think you want a blank line before this so that it doesn't get folded into the previous paragraph when rendered.